akka-actor-typed-tests/src/test/java/jdocs/akka/typed/testing/async/BasicAsyncTestingTest.java

```diff
@@ -52,7 +49,7 @@ public void cleanup() {
   @Test
   public void testVerifyingAResponse() {
     //#test-spawn
-    TestProbe<Pong> probe = new TestProbe<>(system(), testkitSettings());
+    TestProbe<Pong> probe = new TestProbe<>(system());
```

Comment:
I removed this as none of our tests override it, we can add an additional constructor if we want to override in future